cfn-resources/util/util.go

```diff
@@ -38,7 +38,7 @@ import (
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/logger"
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/version"
 	atlasSDK "go.mongodb.org/atlas-sdk/v20231115002/admin"
-	adminLatest "go.mongodb.org/atlas-sdk/v20231115003/admin"
+	adminLatest "go.mongodb.org/atlas-sdk/v20231115007/admin"
```

Comment:
not sure i would call "latest" to the latest instead of just admin,   I would probably do similar to TF:
admin20231115003 "go.mongodb.org/atlas-sdk/v20231115003/admin" (full version in prev. sdks)
"go.mongodb.org/atlas-sdk/v20231115007/admin"  (no alias so it's admin)

Reply:
I don't have a strong opinion here but changing this name would cause quite a bit of changes and it would be a big PR. I would prefer to do it in a different PR just to keep this as focused as possible and avoid possible issues.

Reply:
like Leos idea so we align with how we are handling this in terraform. +1 for separate PR.

Reply:
ok in a different PR